x-pack/legacy/plugins/siem/public/components/fields_browser/field_items.tsx

```diff
@@ -179,11 +177,11 @@ export const getFieldColumns = () => [
     field: 'description',
     name: i18n.DESCRIPTION,
     render: (description: string) => (
-      <EuiToolTip position="top" content={description}>
-        <TruncatableText size="xs" width="390px">
-          {description}
-        </TruncatableText>
-      </EuiToolTip>
+      <TruncatableText>
+        <EuiToolTip position="top" content={description}>
+          <>{description}</>
```

Comment:
are the surrounding `<>` necessary?

Reply:
Oh, yeah. I meant to ask about that. Typescript was barking at me if I didn't include the fragment. Any idea why?